render/renderable_node.go

```diff
@@ -13,67 +13,42 @@ type RenderableNode struct {
 	LabelMinor string        `json:"label_minor,omitempty"` // e.g. "hostname", human-readable, optional
 	Rank       string        `json:"rank"`                  // to help the layout engine
 	Pseudo     bool          `json:"pseudo,omitempty"`      // sort-of a placeholder node, for rendering purposes
-	Adjacency  report.IDList `json:"adjacency,omitempty"`   // Node IDs (in the same topology domain)
 	Origins    report.IDList `json:"origins,omitempty"`     // Core node IDs that contributed information
 
 	report.EdgeMetadata `json:"metadata"` // Numeric sums
-	report.NodeMetadata `json:"-"`        // merged NodeMetadata of the nodes used to build this
+	report.NodeMetadata
```

Comment:
The non-omitted fields of NodeMetadata will get flattened and rendered into the RenderableNode JSON, i.e. you'll get

```json
{
    "label_minor": "whatever",
    "origins": ["a", "b", "c"],
    "adjacency": ["x", "y", "z"]
}
```

Is that your intent?